config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+     resources :books
```

Comment:
You only define an index an a show, resources defines many other routes. You can do 
`resources :books, only: [:show, :index]`

Reply:
Done!.